tools/data_collection/pydantic_models.py

```diff
@@ -56,3 +56,8 @@ class ModelRun(BaseModel):
     memory_footprint: str = Field(
         "N/A", description="Link to the visual chart of memory footprint of the model."
     )
+    fit_in_memory: str = Field(None, description="Whether model fits in SRAM.")
+    peak_sram_usage: float = Field(
+        None,
+        description="What is the peak SRAM usage for a model during its execution phase.",
```

Comment:
Mention what unit of measurement is being used here.

Reply:
Well in conftest.py, there is no compiled_memory_metric. I have removed it. Don't know how it popped here.
Also in the README peak sram usage is mentioned to be in MB. So mentioning the unit is not a necessity here I think.

Reply:
I think it's still useful to mention MB here. I imagine someone who is only handling the database schemas would find it convenient.

Reply:
Also, I'm not sure what you mean by compiled_memory_metric in this comment chain.